### ISSUE TITLE
could there be a way to do this on a windows VM?

### ISSUE THREAD

**Issue by whatsaheart02** [RarogCmex/Guide-Genshin-Impact-On-VM#1]
i am interested in doing this so i can run Genshin Impact on Shadow PC.

**Comment by RarogCmex** [comment]
Of course, there must be a way to do this. I can't help you with that, sadly.
Check how to hide hypervisor on the VMWare or so on.
